src/model/__tests__/__snapshots__/ColumnConfiguration.spec.tsx.snap

```diff
@@ -904,7 +904,62 @@ exports[`ColumnConfiguration component should render all columns 20`] = `
 </DocumentFragment>
 `;
 
-exports[`ColumnConfiguration component should render all columns 21`] = `<DocumentFragment />`;
+exports[`ColumnConfiguration component should render all columns 21`] = `
```

Comment:
Wondering why there is a loading message displayed here. Is there an async call happening in the columns? Can't remember...

Reply:
This is because the VariationView is used [here](https://github.com/ebi-uniprot/uniprot-website/blob/425faa6d2f27720690ff43965adf575a86faebe2/src/model/ColumnConfiguration.tsx#L386). Is it not supposed to be used here?

Reply:
It should just be the table from Variation view but it looks like there's no options for having the table only. Leave it with me, I'll make the change on my PR as it's linked.